src/page-objects/base.ts

```diff
@@ -220,4 +225,39 @@ export default class BasePageObject {
     const elements = await this.browser.$$(selector);
     return Promise.all(elements.map(async element => element.getText()));
   }
+
+  /**
+   * Attaches observer to collect all live updates from the page that can be fetched with page.getLiveAnnouncements().
+   */
+  async initLiveAnnouncementsObserver() {
+    await this.browser.execute(() => {
+      const observer = new MutationObserver(mutationList => {
```

Comment:
Can you move the actual script content into separate folder: https://github.com/cloudscape-design/browser-test-tools/tree/main/src/browser-scripts

This allows for more precise linter config

Reply:
You also need to add a test, similar to this one:
https://github.com/cloudscape-design/browser-test-tools/blob/main/test/page-object.test.js#L102-L111

Reply:
Thanks, will do!